src/main/java/webserver/request/support/QueryParamters.java

```diff
@@ -0,0 +1,33 @@
+package webserver.request.support;
+
+import webserver.exception.InvalidQueryStringException;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class QueryParamters {
```

Comment:
오타가 있어 소소하게 코멘트 하나 달겠습니다~ 🙏 
```suggestion
public class QueryParameters {
```

Reply:
오타가 있었네요...😥 찾아주셔서 감사합니다. 수정하겠습니다.